src/raster/r.object.thickness/r.object.thickness.py

```diff
@@ -99,10 +135,37 @@
 
 gettext.install("grassmods", os.path.join(os.getenv("GISBASE"), "locale"))
 
+# Cleanup
+CLEAN_LAY = []
+DBTABLE = []
+
+
+def cleanup():
+    """Remove temporary maps specified in the global list"""
+    cleanrast = list(reversed(CLEAN_LAY))
+    for rast in cleanrast:
+        gs.run_command(
+            "g.remove", type=["raster", "vector"], name=rast, quiet=True, flags="f"
+        )
+    if len(DBTABLE) > 0:
+        gs.run_command("db.droptable", flags="f", table=DBTABLE[0], quiet=True)
+
+
+def create_unique_name(name):
```

Comment:
There is https://grass.osgeo.org/grass-stable/manuals/libpython/script.html?highlight=append#script.utils.append_uuid